test/screenshot/index.html

```diff
@@ -1,15 +1,24 @@
 <!DOCTYPE html>
 <html>
-  <a href="button">Button</a>
-  <a href='card'>Card</a>
-  <a href="chips">Chips</a>
-  <a href="fab">Fab</a>
-  <a href="floating-label">Floating Label</a>
-  <a href="line-ripple">Line Ripple</a>
-  <a href="material-icon">Material Icon</a>
-  <a href="notched-outline">Notched Outline</a>
-  <a href="text-field">Text Field</a>
-  <a href="text-field/icon">Text Field Icon</a>
-  <a href="text-field/helper-text">Text Field Helper Text</a>
-  <a href="top-app-bar">Top App Bar</a>
+  <head>
+    <style>
```

Comment:
if you're gonna apply styling...follow all our styling best practices. Uses Sass and BEM please